lib/galaxy/managers/tags.py

```diff
@@ -0,0 +1,55 @@
+from typing import (
+    List,
+    Optional,
+)
+
+from pydantic import (
+    BaseModel,
+    Field,
+)
+
+from galaxy.managers.context import ProvidesUserContext
+
+
+class ItemTagsPayload(BaseModel):
+    item_id: str = Field(
```

Comment:
This should be a `EncodedDatabaseIdField`

Reply:
Good catch!